clang/unittests/Parse/ParseHLSLRootSignatureTest.cpp

```diff
@@ -0,0 +1,375 @@
+//=== ParseHLSLRootSignatureTest.cpp - Parse Root Signature tests ---------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
```

Comment:
clang/lib/Parse/ParseHLSLRootSignature.cpp lacks a License statement at the top of its file.
I'm not sure whether every new cpp file needs a declaration of a license? I bet @beanz knows.

Reply:
Every new file should get the license header.